sfaira/commands/templates/single_dataset/{{ cookiecutter.doi_sfaira_repr }}/extra_description.txt

```diff
@@ -0,0 +1,2 @@
+This file can be used to add further custom details for your dataset.
```

Comment:
Let's leave it empty by default so that people do not commit this file or have to delete stuff if they do not want to use it? Alternatively add a sentence below that this message should not be committed?